.travis.yml

```diff
@@ -3,6 +3,9 @@ go:
   - 1.7.6
   - 1.8.3
   - 1.9.2
+branches:
```

Comment:
do you want to land this too?

Reply:
Yup we'd like to build PRs and just the master branch.  Not when you push, that way during PRs it'll reduce the amount of times we test the head of the PR branch from two to just once.

Reply:
Cool, sounds good.